src/functions/marshal_headers_from_sapi.php

```diff
@@ -35,18 +44,14 @@ function marshalHeadersFromSapi(array $server) : array
             }
         }
 
-        if ($value === '') {
-            continue;
-        }
-
         if (strpos($key, 'HTTP_') === 0) {
-            $name = strtr(strtolower(substr($key, 5)), '_', '-');
+            $name = str_replace('_', '-', strtolower(substr($key, 5)));
```

Comment:
Why have you changed this? It's unrelated...

Reply:
there is no any functional change, unrelated, but just to be inline with the other change below